9.6/paper.tex

```diff
@@ -530,6 +531,8 @@ \subsection{Interface to the \trilinos{} \tpetra{} stack}\label{sec:tpetra}
 most noticeably wrappers for the algebraic-multigrid preconditioner \texttt{MueLu}
 and the iterative solvers from \texttt{Belos}.
 However, the wrapped \texttt{Ifpack2} preconditioners can already be used with the iterative solvers of \dealii{}.
+To do this you will need to explicitly specify the vector type parameter,
+e.g.\\ \texttt{SolverCG<LinearAlgebra::TpetraWrappers::Vector<double, MemorySpace::Default> >}
```

Comment:
I wouldn't provide more details here. The release paper is a brief summary, not a guide.